main/src/cgeo/geocaching/enumerations/CacheType.java

```diff
@@ -36,7 +36,7 @@
     GPS_EXHIBIT("gps", "GPS Adventures Exhibit", "72e69af2-7986-4990-afd9-bc16cbbb4ce3", R.string.gps, R.drawable.type_event, "1304", R.drawable.dot_event), // icon missing
     BLOCK_PARTY("block", "Geocaching HQ Block Party", "bc2f3df2-1aab-4601-b2ff-b5091f6c02e3", R.string.block, R.drawable.type_event, "4738", R.drawable.dot_event), // icon missing
     LOCATIONLESS("locationless", "Locationless (Reverse) Cache", "8f6dd7bc-ff39-4997-bd2e-225a0d2adf9d", R.string.locationless, R.drawable.type_virtual, "12", R.drawable.dot_virtual), // icon missing
-    ADVLAB("advlab", "Adventure Lab Cache", "", R.string.advlab, R.drawable.type_advlab, "7777", R.drawable.dot_advlab),
+    ADVLAB("advlab", "Lab Cache", "", R.string.advlab, R.drawable.type_advlab, "7777", R.drawable.dot_advlab),
```

Comment:
Do not use a dummy `wptTypeId` (7777).
See e.g. USER_DEFINED cache type.
Maybe it is better to rename all without "Adventure"?

Reply:
> Do not use a dummy `wptTypeId` (7777).

> See e.g. USER_DEFINED cache type.
> Maybe it is better to rename all without "Adventure"?

Of course, no problem, just let me know your preference, however,  a global change won't do as GC HQ isn't sure how to call this thing either. The application ist called "Adventure Lab".

Can we come to some recommendations as a team as to exactlty how to name this things in all the various places in the code? I am open to the team's decision.

Reply:
Once comment wrt Lab / Adventure Lab: I think GC HQ has rebranded that game. Years ago those were called "Lab Cache" and they were played at Mega / Giga Events. Since about 2 years ago these things are called Adventure Lab Caches. I'd assume they just haven't renamed it system wide (yet)